backend/bamboo/models.py

```diff
@@ -1,29 +1,32 @@
-from datetime import datetime
+from datetime import UTC, datetime
+from typing import Optional
 
-from sqlalchemy import JSON, Boolean, Column, DateTime, Float, ForeignKey, Integer, String, Text
-from sqlalchemy.orm import relationship
+import sqlalchemy as sa
+import sqlalchemy.orm as so
 from werkzeug.security import check_password_hash, generate_password_hash
 
 from bamboo.core.extensions import db
 
 
 class User(db.Model):
-    __tablename__ = "user"
-    id = Column(Integer, primary_key=True)
-    name = Column(String)
-    username = Column(String)
-    password_hash = Column(String)
-    biology = Column(String)
-    introduction = Column(String)
-    created_at = Column(DateTime, default=datetime.utcnow)
-    updated_at = Column(DateTime, default=datetime.utcnow, onupdate=datetime.utcnow)
-    active = Column(Boolean)
-    profile_image_id = Column(Integer, ForeignKey("media.id"))
-    role_id = Column(String, ForeignKey("role.id"))
-
-    profile_image = relationship("Media", foreign_keys=[profile_image_id])
-    role = relationship("Role", back_populates="users")
-    blog_authors = relationship("BlogAuthor", back_populates="author", cascade="all, delete-orphan")
+    id: so.Mapped[int] = so.mapped_column(primary_key=True)
+    name: so.Mapped[str]
+    username: so.Mapped[str]
+    password_hash: so.Mapped[str]
+    bio: so.Mapped[Optional[str]]
+    introduction: so.Mapped[Optional[str]]
+    created_at: so.Mapped[datetime] = so.mapped_column(default=lambda: datetime.now(UTC))
+    updated_at: so.Mapped[datetime] = so.mapped_column(
+        default=lambda: datetime.now(UTC), onupdate=lambda: datetime.now(UTC)
+    )
+    active: so.Mapped[bool]
```

Comment:
@frostming Do we need to set this field defaults to `False` or `True`?

Reply:
False